client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -834,6 +834,22 @@ static Request getSettings(GetSettingsRequest getSettingsRequest) {
         return request;
     }
 
+    static Request getIndex(GetIndexRequest getIndexRequest) {
+        String[] indices = getIndexRequest.indices() == null ? Strings.EMPTY_ARRAY : getIndexRequest.indices();
+
+        String endpoint = endpoint(indices);
+        Request request = new Request(HttpGet.METHOD_NAME, endpoint);
+
+        Params params = new Params(request);
+        params.withIndicesOptions(getIndexRequest.indicesOptions());
+        params.withLocal(getIndexRequest.local());
+        params.withIncludeDefaults(getIndexRequest.includeDefaults());
+        params.withHuman(getIndexRequest.humanReadable());
+        params.withMasterTimeout(getIndexRequest.masterNodeTimeout());
```

Comment:
can you also add this parameter to the corresponding REST spec? We are missing it there I think.

Reply:
I don't understand what you mean by the REST spec? You mean the documentation for `get_index`?

Reply:
https://github.com/elastic/elasticsearch/tree/master/rest-api-spec/src/main/resources/rest-api-spec/api , specifically https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/indices.get.json

Reply:
Thanks! Now I know what this mysterious "spec" is every time you mention it 😀

Reply:
👍 haha I thought I had explained that, sorry for the confusion.